### ISSUE TITLE
Nukes not destroying nearby structures

### ISSUE THREAD

**Issue by vda8** [studio-minus/ppg-bugs-and-requests#950]
**Describe the bug**
For some reason. Nukes don't destroy nearby destructible objects like concrete and brick.

**Clearly describe how to reproduce the bug**
Make a structure out of bricks or concrete.
Place any nuke inside the structure.
Detonate the nuke.

**Mod list**
I don't use mods for betas, as to better see the vanilla features.

**System info (please complete the following information)**
 - Operating system: Windows 10.
 - Game version: Beta 1.23
 - GPU: Nvidia Geforce GTX 165o
 - CPU: Intel I-5

**Additional notes**
![People Playground 22_03_2022 12 17 42](https://user-images.githubusercontent.com/90781996/159471109-80e91d10-2103-4e7d-b540-9f24017b1a6e.png)

**Comment by DindinYT37** [comment]
Probably because they're frozen and explosions kinda ignore frozen things

**Comment by vda8** [comment]
> Probably because they're frozen and explosions kinda ignore frozen things

No, i don't freeze my "destructable" structures. I put them together with the breakable fixed cable thingamajigs.

**Comment by DindinYT37** [comment]
> > Probably because they're frozen and explosions kinda ignore frozen things
> 
> No, i don't freeze my "destructable" structures. I put them together with the breakable fixed cable thingamajigs.

Could also just be out of range.

**Comment by vda8** [comment]
> > > Probably because they're frozen and explosions kinda ignore frozen things
> > 
> > 
> > No, i don't freeze my "destructable" structures. I put them together with the breakable fixed cable thingamajigs.
> 
> Could also just be out of range.

not out of range. the nuke is detonated inside the structure.

**Comment by DindinYT37** [comment]
> > > > Probably because they're frozen and explosions kinda ignore frozen things
> > > 
> > > 
> > > No, i don't freeze my "destructable" structures. I put them together with the breakable fixed cable thingamajigs.
> > 
> > 
> > Could also just be out of range.
> 
> not out of range. the nuke is detonated inside the structure.

Then I don't know, dark magic perhaps?

**Comment by Gigi-7688** [comment]
in a realistic context, anything within a certain distance of the explosion should be vaporized instantly

**Comment by Gigi-7688** [comment]
And any exposed object (wood, any life form except robots) should immediately catch fire

**Comment by vda8** [comment]
> > > > > Probably because they're frozen and explosions kinda ignore frozen things
> > > > 
> > > > 
> > > > No, i don't freeze my "destructable" structures. I put them together with the breakable fixed cable thingamajigs.
> > > 
> > > 
> > > Could also just be out of range.
> > 
> > 
> > not out of range. the nuke is detonated inside the structure.
> 
> Then I don't know, dark magic perhaps?

maybe
